### PR TITLE
chore: update CI references to use SHA versions

### PR DESCRIPTION
**Pull request by kevmoo**
And add dependabot configuration

### CODE REVIEW

**Review by Lyokone — APPROVED** [review]

**Comment by kevmoo** [comment]
I don't have right to merge.

**Comment by Lyokone** [comment]
I'm waiting for another approval to merge @russellwheatley @lesnitsky

**Review by russellwheatley — APPROVED** [review]

**Pull request merged**
